src/docs/what-is-polywrap.md

```diff
@@ -0,0 +1,74 @@
+---
+id: what-is-polywrap
+title: What is Polywrap
```

Comment:
This seems strange to have when we already have an introduction

Reply:
I kept it as such, as it's a copy of the more detailed explanation from our Blog, and it was the plan within the Notion.

I'd wager we can remove this? Or maybe turn it into a "Web3 use-case" kind of doc?